src/utils/session-core.js

```diff
@@ -683,33 +685,52 @@ export const updateUserActivity = async (session) => {
  * @function sendMessageTTL
  * @param {Session} session - Solid's Session Object {@link Session}
  * @param {object} messageObject - An object containing inputs for the the message
- * @param {string} otherPodUsername - The username of the other user you're sending a message to
  * @returns {Promise} Promise - Updates last active time of user to lastActive.ttl
```

Comment:
Not added by this PR, but this return value is incorrect. This function has no return statement; it returns nothing.

Reply:
Ah ok. When I was debugging, this function didn't return anything, but that was actually because of the `forEach` statement at the end, which you've fixed.